### ISSUE TITLE
How do we handle sprint review?

### ISSUE THREAD

**Issue by Almantask** [csinn/CSInn#29]
At the end of each sprint we need to have a retrospective review. So every 2 weeks (that's the sprint length) we need to answer the following questions:

-  What was the goal of sprint?
- Did we manage to achieve our goals?
- What struggles did we have?
- How do people feel about the project?
- What could be improved?

Would you like this to be within an issue or would you like to gather on a discord voice chat to discuss it?

**Comment by Almantask** [comment]
We need to modify our way of work a bit, to better adjust to everyone. We are not doing agile, simply because we don't work enough, which is understandable. I think every two weeks, having a simple Q&A to the questions above will do.

**Comment by EmKa-dev** [comment]
I agree, a simple q&a will do. Maybe we can also ask around to get a picture of where everyone is at, and what they think.
